web/js/image/image.panel.js

```diff
@@ -259,10 +264,9 @@ wv.image.panel = wv.image.panel || function (models, ui, config) {
 
       // get layer transparencies (opacities)
       var opacities = [];
-      _(products)
```

Comment:
Yes, this tripped me up at first because the previous syntax is wrong. I pulled down a version of development to double-check that correcting this syntax didn't break anything else.